lms/djangoapps/courseware/features/lti.py

```diff
@@ -129,7 +132,7 @@ def incorrect_lti_is_rendered(_step):
 
 @step('the course has correct LTI credentials with registered (.*)$')
 def set_correct_lti_passport(_step, user='Instructor'):
-    coursenum = 'test_course'
+    coursenum = TEST_COURSE_NAME
```

Comment:
You mean `coursenum` ? Good point. 

Unfo that's also what it's called in the method, too: https://github.com/edx/edx-platform/blob/benp/lms-one-shard/lms/djangoapps/courseware/features/lti.py#L236

And yeah...that's confusing. The file also has methods that are the same name (with different signatures) that exist in the common package helper lettuce files :(